scripts/run_ec2.sh

```diff
@@ -0,0 +1,22 @@
+# !bin/bash
+MAGE_CLI="python3 -m mage_ai.command_line"
```

Comment:
You should be able to just use `mage init` and `mage start`. Any specific reason of using the command_line file?

Reply:
I thought so too, but executing these commands through a bash script never seemed to work for me - the script always threw an error saying that no command `mage` existed _despite_ being able to use the same exact command when logged on manually to the EC2 instance. My workaround was to manually target the python command line file.

Reply:
ok